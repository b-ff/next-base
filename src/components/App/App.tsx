import { SessionProvider } from "next-auth/react";
import { FC, ReactNode } from "react";

export type AppProps = {
  children: ReactNode;
};

export const App: FC<AppProps> = ({ children }) => (
  <SessionProvider>{children}</SessionProvider>
);
