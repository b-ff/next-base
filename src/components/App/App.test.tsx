import { App } from "./App";
import { render } from "@testing-library/react";

const mockChildren = "content";

describe("Component APP", () => {
  it("Should render as expected", () => {
    const { container } = render(<App>{mockChildren}</App>);
    expect(container).toMatchSnapshot();
  });

  it("Should render given children", () => {
    const dom = render(<App>{mockChildren}</App>);
    const initials = dom.getByText(mockChildren);
    expect(initials).not.toBeNull();
  });
});
