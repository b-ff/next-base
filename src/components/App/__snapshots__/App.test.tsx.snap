// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component APP Should render as expected 1`] = `
<div>
  <div>
    content
  </div>
</div>
`;
